packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -268,6 +284,11 @@ module.exports = {
     new HtmlWebpackPlugin({
       inject: true,
       template: paths.appHtml,
+      // Need to ensure that the runtime is loaded first because
+      // it loads the rest, then the vendors
+      // And then the polyfills before the main
+      // The others chunks will be loaded after
+      chunksSortMode: sortChunks(['runtime', 'vendors', 'polyfills', 'main']),
```

Comment:
Why don't we remove this since we said that it automatically detects the correct order?
Let's not add this until a problem arises (if one does).